packages/@lwc/engine-server/src/apis/render-component.ts

```diff
@@ -30,8 +30,9 @@ const FakeRootElement: HostElement = {
 export function renderComponent(
     tagName: string,
     Ctor: typeof LightningElement,
-    props: { [name: string]: any } = {}
-): string {
+    props: { [name: string]: any } = {},
+    options: SsrOptions = {}
+): { html: string; styles: string[] } {
```

Comment:
Would be good to have the return value be a type we expose.

Reply:
Yes, that should be an easy fix if we want it.